src/Web/Slack/Types/Message.hs

```diff
@@ -1,28 +1,33 @@
 {-# LANGUAGE DeriveGeneric     #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TemplateHaskell   #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
 
 module Web.Slack.Types.Message where
 
-import Control.Lens.TH
-import Data.Aeson
-import Data.Aeson.TH
-import Data.Aeson.Types
-import Data.Maybe
-import qualified Data.Text as T
-import Data.Time.Clock.POSIX
-import GHC.Generics
-import Web.Slack.Types.Base
-import Web.Slack.Types.Id
-import Web.Slack.Types.Time
-import Web.Slack.Utils
+import Control.Lens (makeLenses,(^.))
+import Data.Aeson.TH (deriveToJSON, deriveFromJSON, defaultOptions,fieldLabelModifier)
+import Data.Aeson.Types (FromJSON(..), Object, Parser, ToJSON(..), Value (..), KeyValue,
```

Comment:
I just can't read another's code without it(i should google every unknown function, instead of just reading doc for module with it, also functions often have ambigous names etc). 
That is why i really understand the meaning ot that.  

:-)